Index: app/src/main/java/com/seoyoon/safeproject/MyPageActivity.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.seoyoon.safeproject\n\nimport androidx.appcompat.app.AppCompatActivity\nimport android.os.Bundle\n\nclass MyPageActivity : AppCompatActivity() {\n    override fun onCreate(savedInstanceState: Bundle?) {\n        super.onCreate(savedInstanceState)\n        setContentView(R.layout.activity_my_page)\n    }\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/seoyoon/safeproject/MyPageActivity.kt b/app/src/main/java/com/seoyoon/safeproject/MyPageActivity.kt
--- a/app/src/main/java/com/seoyoon/safeproject/MyPageActivity.kt	(revision bf44518dd962d4efc9966dceb8b06edf679b98ac)
+++ b/app/src/main/java/com/seoyoon/safeproject/MyPageActivity.kt	(date 1699678179820)
@@ -6,6 +6,6 @@
 class MyPageActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_my_page)
+        setContentView(R.layout.activity_mypage)
     }
 }
\ No newline at end of file
